xt/66-cucumber/01-basic/preferences.feature

```diff
@@ -0,0 +1,72 @@
+# Using one db would be quicker but the menu language would persist from
+# previous pass and PageObject/App/Menu.pm would have to be able to handle
+# non-english menus
+@one-db @weasel
+Feature: Check correct operation of Preferences screen
+
+Background:
+  Given a standard test company
+    And a logged in admin
+
+Scenario: I change user preferences to the <selection> language
+  When I navigate the menu and select the item at "Preferences"
+   And I select the "Preferences" tab
+   And I select "<selection>" from the drop down "Language"
+   And I save the page
+   And I select the "Preferences" tab
+  Then I expect "<translation>" to be selected for "<language>"
+  When I select "<default>" from the drop down "<language>"
+   And I click "<save>" to save the page
+   And I select the "Preferences" tab
+  Then I expect "English" to be selected for "Language"
+
+# Non UTF8 accented characters are confirmed working
```

Comment:
No need to put this in comments: `Examples` should be able to have descriptions: https://github.com/cucumber/cucumber/blob/master/gherkin/testdata/good/descriptions.feature#L50

Reply:
Fixed